notify/impl.go

```diff
@@ -1551,6 +1558,21 @@ func (a *loginAuth) Start(server *smtp.ServerInfo) (string, []byte, error) {
 	return "LOGIN", []byte{}, nil
 }
 
+type noAuth struct{}
```

Comment:
Please move this after `func (a *loginAuth) Next(fromServer []byte, more bool) ([]byte, error) {...}`

Reply:
Hah, and there i was wondering why `loginAuth` didnt have a `Next` interface